### PR TITLE
style(ui): Tune prettier to align with the rest of the codebase

### PR DESCRIPTION
**Pull request by Etsija**
Please have a look at the individual commits for details.

### CODE REVIEW

File: ui/.prettierrc [hunk 1]
```diff
@@ -3,5 +3,8 @@
   "jsxSingleQuote": true,
   "trailingComma": "es5",
   "useTabs": false,
-  "plugins": ["@ianvs/prettier-plugin-sort-imports"]
+  "plugins": [
```
Comment (sschuberth) [inline ui/.prettierrc:6]: Commit message nit: We do not use a double-space to separate sentences in a paragraph.

Comment (Etsija) [reply ui/.prettierrc:6]: Sry, it still comes from flesh memory from time to time...fixed now.

**Comment by mmurto** [comment]
It could make sense to [configure separation between import groups ](https://github.com/trivago/prettier-plugin-sort-imports?tab=readme-ov-file#importorderseparation)to align with Kotlin code.

**Comment by Etsija** [comment]
> It could make sense to [configure separation between import groups ](https://github.com/trivago/prettier-plugin-sort-imports?tab=readme-ov-file#importorderseparation)to align with Kotlin code.

I don't think the plugin I'm using supports that.

**Comment by mmurto** [comment]
> > It could make sense to [configure separation between import groups ](https://github.com/trivago/prettier-plugin-sort-imports?tab=readme-ov-file#importorderseparation)to align with Kotlin code.
> 
> I don't think the plugin I'm using supports that.

Oh, didn't notice it was different, trivago's seems to be more popular. [But your choice supports is as well](https://github.com/IanVS/prettier-plugin-sort-imports?tab=readme-ov-file#3-add-spaces-between-import-groups).

**Comment by Etsija** [comment]
> > > It could make sense to [configure separation between import groups ](https://github.com/trivago/prettier-plugin-sort-imports?tab=readme-ov-file#importorderseparation)to align with Kotlin code.
> > 
> > 
> > I don't think the plugin I'm using supports that.
> 
> Oh, didn't notice it was different, trivago's seems to be more popular. [But your choice supports is as well](https://github.com/IanVS/prettier-plugin-sort-imports?tab=readme-ov-file#3-add-spaces-between-import-groups).

Yes, but then we need to invent an import order instead of using the default one, which seems to work well. I'm open to suggestions as to the definition of the import order.

**Comment by mmurto** [comment]
> Yes, but then we need to invent an import order instead of using the default one, which seems to work well. I'm open to suggestions as to the definition of the import order.

A good start would be the default but with lines between. :)

File: ui/src/components/providers.tsx [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
  */
 
+import { OpenAPI as OpenAPIConfig } from '@/api/requests/core/OpenAPI';
+import { Toaster } from '@/components/ui/toaster';
 import { queryClient } from '@/lib/query-client';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactNode } from 'react';
 import { AuthProvider, AuthProviderProps } from 'react-oidc-context';
-import { Toaster } from '@/components/ui/toaster';
-import { OpenAPI as OpenAPIConfig } from '@/api/requests/core/OpenAPI';
```
Comment (mmurto) [inline ui/src/components/providers.tsx:25]: Just an example but this happens across the files. The sorting does not work, as it considers the imports from local code with the alias `@/` to be third-party code. Not sure what the best way to fix it is, but the differentiator is that third party code from namespaces doesn't have `/` after `@`.

**Review by mmurto — CHANGES_REQUESTED** [review]

**Review by mmurto — APPROVED** [review]

File: ui/.prettierrc [hunk 3]
```diff
@@ -2,5 +2,14 @@
   "singleQuote": true,
   "jsxSingleQuote": true,
   "trailingComma": "es5",
-  "useTabs": false
```
Comment (MarcelBochtler) [inline ui/.prettierrc:4]: Commit message:
```
To align the better with the rest
```

This should probably be:
```
To align better with the rest
```

**Pull request merged**